src/molecules/formfields/CreditCardField/Readme.md

```diff
@@ -1,5 +1,27 @@
 CreditCardField Example:
 
 ```example
-    <CreditCardField />
+    <CreditCardField>
+      <CreditCardField.Top>
+        <TextField
+          placeholder='Credit Card Number'
+          noBaseStyle
+        />
+      </CreditCardField.Top>
+
+      <CreditCardField.Bottom>
+        <TextField
+          placeholder='Expiration Date'
+          input={{
+            pattern: '[0-9]{13,16}'
```

Comment:
hmm, is this necessary though? Do we need this plus validations, it seems redundant. also seems like someone could just delete this off the DOM and input whatever they like.